### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by dominikhorn93**

### CODE REVIEW

**Comment by dominikhorn93** [comment]
@boal any suggestions to add more information to the README ?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+_Below is an example of how you can installing and setup up your service_
+
+1. Use the spring initalizer and areate a Spring Boot application with `Spring Web` dependencies [https://start.spring.io](https://start.spring.io)
```
Comment (boal) [inline README.md:90]: typo ```areate```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+With Gradle:
+```
+implementation group: 'io.muenchendigital.digiwf', name: 'digiwf-s3-integration-starter', version: '${digiwf.version}'
+```
+3. Configure your S3 bucket with the following properties:
+    - `io.muenchendigital.digiwf.s3.bucketName`
+   - `io.muenchendigital.digiwf.s3.secretKey`
+   - `io.muenchendigital.digiwf.s3.accessKey`
+   - `io.muenchendigital.digiwf.s3.url`
+
```
Comment (boal) [inline README.md:110]: Adding property description for cronjob.

```io.muenchendigital.digiwf.s3.cleanupcronjob```

Comment (boal) [reply README.md:110]: or reference to lower section ```Cron Job Cleanup```

**Review by boal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dominikhorn93) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (dominikhorn93) [reply README.md:112]: done with reference

**Review by boal — APPROVED** [review]

**Pull request merged**